src/commands.rs

```diff
@@ -1067,12 +1067,18 @@ fn cmd_init(ui: &mut Ui, command: &CommandHelper, args: &InitArgs) -> Result<(),
         ));
     }
     let wc_path = ui.cwd().join(&args.destination);
-    if wc_path.exists() {
-        assert!(wc_path.is_dir());
-    } else {
-        fs::create_dir(&wc_path).unwrap();
+    match fs::create_dir(&wc_path) {
+        Ok(()) => {}
+        Err(_) if wc_path.is_dir() => {}
+        Err(e) => {
+            return Err(UserError(format!(
+                "Failed to create working directory: {e}"
```

Comment:
I'm worried people will think of "working directory" as `$PWD`. What do you think about "workspace"? That's what we use internally for the combination of a working copy and repo, and this command creates both, so that sounds reasonable. (I'm open to changing the terminology, btw.)

Reply:
Sure, shorter consistent name is better. Updated.